rest.md

```diff
@@ -106,17 +106,41 @@ Create routes for the following requests. The first one is done for you.
 1. Create a new animal
   - `POST /animals`
 2. Delete an animal
```

Comment:
Very small formatting thing, but it might make the lesson plan more readable if you make each numbered item bold (e.g., 2. **Delete an animal**). You can leave your details-summary blocks as they are.